LEGEND.md

```diff
@@ -4,48 +4,88 @@
 
 [2. Slide Shortcuts](#slides)
 
+- [macOS](#macos-slides)
+
+- [Windows](#windows-slides)
+
 [3. Others](#others)
 
+- [macOS](#macos-others)
+
+- [Windows](#windows-others)
+
 ### <a name="mappings">Keyboard Mappings</a>
 
 ![](./assets/img/help_images/web-desktop-keyboard-map.png)
 
-- _Note that the chars on the top require shift + key._
+- _Note that the characters shown at the top of each key require a shift + key._
 
 ### <a name="slides">Slide Shortcuts</a>
 
-#### macOS:
-
-- Cmd + 1 = Waheguru Slide
-- Cmd + 2 = Mool Mantra Slide
-- Cmd + 3 = Empty Slide
-- Cmd + 4 = Anand Sahib (Bhog)
-- Cmd + 5 = Help Guide
-- Cmd + 6 = Shortcut Legend
-- Cmd + / = Jump to search
-
-#### Windows:
-
-- Ctrl + 1 = Waheguru Slide
-- Ctrl + 2 = Mool Mantra Slide
-- Ctrl + 3 = Empty Slide
-- Ctrl + 4 = Anand Sahib (Bhog)
-- Ctrl + 5 = Help Guide
-- Ctrl + 6 = Shortcut Legend
-- Cmd + / = Jump to search
+#### <a name="macos-slides">macOS:</a>:
+
+| Combination | Description                     |
+| ----------- | ------------------------------- |
+| Fn + 1     | Waheguru Slide                  |
+| Fn + 2     | Mool Mantra Slide               |
+| Fn + 3     | Empty Slide                     |
+| Fn + 4     | Anand Sahib (Bhog)              |
+| Fn + 5     | Help Guide                      |
+| Fn + 6     | Shortcut Legend                 |
+| Cmd + /     | Jump to search bar              |
+| ----------- | -----------                     |
+| cmd + 1      | Dhan Guru Nanak Dev Ji          |
+| cmd + 2      | Dhan Guru Angad Dev Ji          |
+| cmd + 3      | Dhan Guru Amar Das Ji           |
+| cmd + 4      | Dhan Guru Raam Das Ji           |
+| cmd + 5      | Dhan Guru Arjun Dev Ji          |
+| cmd + 6      | Dhan Guru Hargobind Sahib Ji    |
+| cmd + 7      | Dhan Guru Har Rai Sahib Ji      |
+| cmd + 8      | Dhan Guru Har Krishan Sahib Ji  |
+| cmd + 9      | Dhan Guru Teg Bahadur Sahib Ji  |
+| cmd + 0      | Dhan Guru Gobind Singh Sahib Ji |
+| cmd + -      | Dhan Guru Granth Sahib Ji       |
+
+#### <a name="windows-slides">Windows:</a>
+
+| Combination | Description                     |
+| ----------- | ------------------------------- |
+| fn + 1    | Waheguru Slide                  |
```

Comment:
will this be an issue for windows keyboards that don't have a fn key?

Reply:
Probably